test/units/testsuite-73.sh

```diff
@@ -656,11 +656,18 @@ EOF
 export SYSTEMD_KBD_MODEL_MAP=/usr/lib/systemd/tests/testdata/test-keymap-util/kbd-model-map
 
 enable_debug
-test_locale
-test_vc_keymap
-test_x11_keymap
-test_convert
-test_validate
+
+# Create a list of all functions prefixed with testcase_
```

Comment:
This looks like something that should be wrapped into a function (`run_testcases`?), moved into `util.sh` and used in other tests too (`TEST-35-LOGIN` comes to mind).

Reply:
`test_` is way too generic and might match possible test-related helpers (if introduced in the future).